config/locales/admin/en/admin.yml

```diff
@@ -310,6 +310,9 @@ en:
     preview:
       no_custom_meditations: "The custom meditation interface cannot be previewed, and so it has been hidden for this preview."
 
+    slugs:
+      previous_slugs: "Previous URLs Used"
```

Comment:
Can you move this translation line to `misc.previous_slugs` instead of putting it in it's own block, and can you also add it all the other `admin.yml` files. You don't need to translate it, just add the entry.